cocos/ui/UIEditBox/iOS/CCUISingleLineTextField.mm

```diff
@@ -45,4 +44,47 @@ - (CGRect)editingRectForBounds:(CGRect)bounds
     return [self textRectForBounds:bounds];
 }
 
+- (void)setInnerPadding:(float)left :(float)top :(float)right :(float)bottom
+{
+    auto glview = cocos2d::Director::getInstance()->getOpenGLView();
+    float factor = glview->getScaleX() / glview->getContentScaleFactor();
+    
+    _leftPadding = left * factor;
+    _topPadding = top * factor;
+    _rightPadding = right * factor;
+    _bottomPadding = bottom * factor;
+    
+}
+
+- (void)setTextAlignMent:(cocos2d::TextHAlignment)hAlign :(cocos2d::TextVAlignment)vAlign
```

Comment:
Sorry This is typo mistake I will fix it. Can I need to send a new PR for this.Please explain me because this is my first PR. Can I make change and update it in this PR?